source/more-info/unhealthy/privileged.markdown

```diff
@@ -14,5 +14,9 @@ If you are running an older version of our Home Assistant OS, update it in the
 {% my configuration title="Configuration" %} panel.
 
 If this is not our Home Assistant OS, your operating system might be out of date. Try checking for and
-installing updates, then restarting your system. If this doesn't work, you may need to re-run our
-[convenience installation script](https://github.com/home-assistant/supervised-installer).
+installing updates, then _restarting your system_.
+
+If this doesn't work, you may need to re-run our
+[convenience installation script](https://github.com/home-assistant/supervised-installer). Note this 
+may lock you out of your instance unless you have physical access, since it will reset all
+network settings.
```

Comment:
This is also no longer a install script actually, so we probably should suggest this:

```suggestion
If this doesn't work, you may need to re-install the [Home Assistant Supervised Debian Package](https://github.com/home-assistant/supervised-installer).
```

@ikifar2012 more generally, do we have instructions how to update/maintain a Supervised installation? What is the general expectation?

Reply:
The expectation is that the Debian package and OS Agent are supposed to always be on the latest version... it seems the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) mentions the agent always being up to date but not the installer package itself, maybe it should explicitly mention that in the ADR?